### DISCUSSION TITLE
XDEFI Wallet Integration

### DISCUSSION THREAD

**Discussion by davidp94** [rainbow-me/rainbowkit#428] [Ideas]
Dear Rainbowkit Team,
We are big fans of your product which offers great added value to Web3 developers! We see more and more protocols implementing your wallet connect kit and would be excited to be part of it.  XDEFI Wallet  ([xdefi.io](http://xdefi.io/)) is a multi-chain web3 browser extension wallet dedicated to DeFi & NFTs - currently supporting 11 different blockchains including Bitcoin, Avalanche, Polygon, Ethereum, Terra and BSC.
XDEFI Wallet has currently more than 80,000 active users with a five star rating on Google’s Chrome Store. We did an overview of popular browser extension wallets in this article that gives you more background on how we differentiate ourselves: https://www.xdefi.io/best-web3-wallet-extension-defi-nft/
We would love to work with you to get XDEFI Wallet listed as a default option within your ‘connect wallet’ kit. We have a well-documented SDK for it, ready-made examples limiting the necessary development effort.
Is this something you would be open to talk about?

Cheers
The XDEFI Wallet Team

PS: Should we just submit a PR?

**Comment by nickbytes** [comment]
Hey @davidp94 – thanks for the kind words! We're welcoming any PRs to add new connectors to the RainbowKit package, but for the time being, we will not be adding anymore wallets to our `getDefaultWallets` helper function. 

One thing you may have noticed is that this function is just a shorthand way to add a group of specific wallets + a heading for that grouping. You could create a new grouping of your own choosing, or even create a package that has your favorite wallets grouped however you would like – we want to push the community to do more of this vs relying on any opinions from us.

**Reply by archon0x** [reply]
Hey @nickbytes, I just wanted to let you know that XDEFI Wallet has grown by 50%, supporting more than 120,000 weekly active users. We also added Arbitrum, Avalanche, and Solana as supported blockchains (with NEAR, Cosmos, and Tron following soon). We would love to discuss again integrating us into your getdefaultwallets helper function. Are you open for that?
